common/lib/xmodule/xmodule/tests/test_conditional.py

```diff
@@ -72,7 +72,7 @@ def create(system, source_is_error_module=False, source_visible_to_staff_only=Fa
                                             "problem", "SampleProblem", deprecated=True)
         if source_is_error_module:
             # Make an error descriptor and module
```

Comment:
nit: @symbolist I am not sure if we should update wording from `descriptor` to `block`?